bookkeeper-server/src/main/java/org/apache/bookkeeper/tls/TLSContextFactory.java

```diff
@@ -58,7 +58,20 @@
      * Supported Key File Types.
      */
     public enum KeyStoreType {
-        PKCS12, JKS, PEM;
+        PKCS12("PKCS12"),
```

Comment:
I don't think we need this change here. If I understand enum correctly, if you print enum, it will use its text literal as toString.